client/src/components/Navbar.jsx

```diff
@@ -1,35 +1,179 @@
-import { AppBar, Box, Link, Stack } from "@mui/material";
-import React, { useEffect } from "react";
+import React, { useContext, useState } from "react";
+import {
+  AppBar,
+  Box,
+  Grid,
+  Link,
+  Typography,
+  MenuItem,
+  Popover,
+  IconButton,
+  Button,
+} from "@mui/material";
+import MenuIcon from "@mui/icons-material/Menu";
+import TrackerIcon from "../assets/tracker_icon.svg";
+import AccountIcon from "../assets/account_icon.svg";
+import LogoPath from "../assets/internship_tracker_logo.svg";
+import { AuthContext } from "../App";
 
-import bobaTalksLogoPath from "../assets/bobatalks_wide.svg";
+const pageNames = ["Find an internship", "About"];
```

Comment:
This would be better as an object or an array of objects rather than two arrays of strings

Reply:
if we use an object we'll be able to also use those entries for breadcrumbs later too